doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -7,27 +7,54 @@
 * [4.1 Introduction](#4.1)
 * [4.2 Prerequisites](#4.2)
 * [4.3 Installation of the Reference Implementation](#4.3)
-  * [4.3.1 Placeholder for Installer-A](#4.3.1)
-  * [4.3.2 Placeholder for Installer-B](#4.3.2)
+  * [4.3.1 Host Provisioning](#4.3.1)
+  * [4.3.2 Software Provisioning](#4.3.2)
```

Comment:
In RI-2 Chapter 3, this is referred to as Kubernetes provisioning. the change can be made in RI2-Chapter 03 or in this chapter

Reply:
Agreed. Kubernetes provisioning seems a better term.

Reply:
@rihabbanday Would you be changing the section headers to "Kubernetes Provisioning"?

Reply:
@pgoyal01 Done